### PR TITLE
1letter/bugfix master

### LINKED ISSUE #37: Line ending-related test failures under Python 2.7
It looks like #36 was never tested under Python 2.7 so now there are line ending-related failures. Travis CI is broken for us right now, but I would expect anyone creating a PR to run `bin/tox` in the development sandbox to make sure all supported Python versions are tested.

@1letter Please revisit the code on the `master` branch and fix those test failures. You can run all tests in your sandbox with `bin/tox`, or just the Python 2.7 tests with `bin/tox -epy27`.

### PR DESCRIPTION
**Pull request by 1letter**
@dataflake: I have updated the tests. I run now bin/tox and all test in all envs are green.

Fixes #37.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -8,6 +8,9 @@ Changelog
   This may require adaptations in testsSupport messages with line separation ``\r\n``
   (`#35 <https://github.com/zopefoundation/Products.MailHost/issues/35>`_).
 
+- Bugfix Tests
+  (`#37 <https://github.com/zopefoundation/Products.MailHost/issues/37>`_).
+
```
Comment (icemac) [inline CHANGES.rst:13]: I think this entry is not necessary because you are fixing tests which where broken after the last release.

**Review by icemac — APPROVED** [review]
LGTM.

**Comment by icemac** [comment]
BTW I started on working to get GitHub Actions integration for the packages using `zc.buildout` in `tox.ini`. But this is way more complex than the packages using `pip`. Let's see when I find the time to present a first working version.

**Review by mauritsvanrees — APPROVED** [review]
LGTM.
I confirm the tox tests are green locally.
(Except the flake8 tests, but maybe I have some global config file that causes this. It also reports import order changes in `src/Products/MailHost/exportimport.py`, which has not been touched since almost two years. So seems safe to ignore, especially since they [pass for 1letter](https://github.com/zopefoundation/Products.MailHost/issues/37#issuecomment-763383893))
Fine to merge.

**Comment by jugmac00** [comment]
Just a note:
For Zope projects we usually create a branch within the project, so the checkout for the reviewer is easier (without adding a new upstream).

**Comment by mauritsvanrees** [comment]
> Just a note:
> For Zope projects we usually create a branch within the project, so the checkout for the reviewer is easier (without adding a new upstream).

This is not a new upstream. The branch is right here in the zopefoundation repo:
https://github.com/zopefoundation/Products.MailHost/tree/1letter/bugfix-master

I guess the `1letter/bugfix-master` name with slash makes it look like an upstream repo, but it is not. I myself often make a branch called `maurits/some-fix`.

**Comment by jugmac00** [comment]
Thanks @mauritsvanrees! There is no day without learning something new!

**Comment by icemac** [comment]
Released to PyPI as https://pypi.org/project/Products.MailHost/4.11/.

**Pull request merged**